invenio_cli/helpers/commands.py

```diff
@@ -151,6 +151,23 @@ def _ensure_containers_running(self):
         docker_helper.start_containers()
         # TODO: Find faster way to procede when containers are ready
         time.sleep(30)  # Give time to the containers to start properly
+    
+    def destroy(self):
+        """Destroys env, containers."""
+        try:
+            subprocess.run(['pipenv', '--rm'], check=True)
+            click.secho('Virtual environment destroyed', fg='green')
+
+        except:
+            pass
```

Comment:
major: We agreed IRL that this function call makes the CLI abort if the venv was not created by pipenv. So we need to catch it. However, it is not a good practice to simply `except` because you can be catching *any other* exception and the CLI will silently fail (this most likely ends-up in many hours of debugging and head aches). 

In addition, `pass` might sometimes be needed. But I would say *try to avoid it, because it looks fishy - you are hiding something*. In this case, I would suggest to use click to display a message to the user to inform of what happened: "The virtual environment was not removed as it was not created by pipenv". This way, first the user knows whats happening, and if something goes wrong it's easier for him to notify us :)

Therefore:
1- `except` just the type of exception needed
2- Display a message in the except.

Reply:
This https://github.com/inveniosoftware/invenio-cli/pull/148/commits/3e9dec8500cbd8022a148f978db66ae1465139b4  solves it :)